fmriprep/cli/run.py

```diff
@@ -166,6 +166,7 @@ def main():
         write_derivative_description(
             config.execution.bids_dir, config.execution.output_dir / "fmriprep"
         )
+        write_bidsignore(config.execution.output_dir / "fmriprep")
```

Comment:
should we add another `.bidsignore` at the top level containing `freesurfer/`? Or at least when `--fs-subjects-dir` was not specified?

Reply:
The directory containing FreeSurfer is not a BIDS dataset.